pkg/cstor/cstor.go

```diff
@@ -536,3 +506,16 @@ func (p *Plugin) getVolInfo(volumeID, snapName string) (*Volume, error) {
 
 	return vol, nil
 }
+
+// getScheduleName return the schedule name for the given backup
+func (p *Plugin) getScheduleName(backupName string) string {
+	// for non-scheduled backup, we are considering backup name as schedule name only
+	scheduleName := backupName
+
+	// If it is scheduled backup then we need to get the schedule name
+	splitName := strings.Split(backupName, "-")
```

Comment:
what happens if non-scheduled backupname consists of multiple '-' in it? is that allowed?

Reply:
>  is that allowed?

Yes, we can create a backup with multiple `-` in it. For example:- `mongodb-backup-1`

Reply:
updated the logic to fetch schedule name

Reply:
Thanks for the update @mynktl

Reply:
thanks @mynktl .. probably we will add a documentation with naming limitation of this 'timestamp' type at the end

Reply:
@vishnuitta I've updated README with this details. https://github.com/openebs/velero-plugin/pull/57/commits/f6a6c7b92c186082a2c51b07b42915d060a219e8#diff-04c6e90faac2675aa89e2176d2eec7d8R113